### DISCUSSION TITLE
Feedback: Some issues I have with osu!lazer

### DISCUSSION THREAD

**Discussion by crux-03** [ppy/osu#22027] [Ideas]
So for starters, I've been testing out osu lazer for the past week and a half or so, totaling in about 20-30 hours. In my opinion, the game feels great to play for the most part, though I still have some issues and "complaints".
- **Certain skins cause severe amounts of lag**
When selecting some skins, the game would begin to stutter. I was playing at a relatively low end computer at the time, but there was a clear discrepancy between stable and lazer performance.
[The skin in question](https://drive.google.com/file/d/1HBARkiLMqDd9Xo-6x07-JGTMjFNC1f3x/view?usp=sharing)
- **Cluttered and unclear UI**
This is very much a personal preference, but in my experience, the UI can feel pretty clunky and almost confusing to navigate at times. It doesn't have the same flow as the stable UI, and I'm missing a lot of the simplicity from stable. Most notably the mod selection and the icons at the top (rankings, beatmap listing etc.) can be hard to navigate, in the case of the icons, I feel that they all look too much alike, and some may not be relevant to have there in the first place (such as news and forums). There is also a minor issue with the beatmap listing, where you have to remove the "Featured Artists" filter every time you enter the page, as it seems to reset every time you restart the game.
- **Lack of a proper quick restart**
This is a feature I'm sure quite a few people are missing. The CTRL + R feature from stable. I mean, it's there in form of the "Quick retry" setting, but it feels slower and more obstructive. The most common use case for this feature is to "retry-spam" maps for better runs. Lazer's loading screen feels obstructive, and takes away from that "quick restart" feel. I personally feel that it can draw the player's attention away from the game, and result in a less enjoyable experience.
[Have a comparison](https://youtu.be/inGsAO3PVFY)
- **Ability to save replays in multiplayer**
This is a minor one, yet one that makes a big difference for me switching to lazer. Currently, there is no ability to save replays at the end of multiplayer maps, which is something I tend to use often.

**Comment by mk56-spn** [comment]
1. the issue with some skins being laggy is known, usually happens on unnecesarily large skins ( 70 mb + ) from what little testing ive done, which this seems to be 

2. > "There is also a minor issue with the beatmap listing, where you have to remove the "Featured Artists" filter every time you enter the page, as it seems to reset every time you restart the game."

this was  an intentional change made due to the FA program now being mature, as ppy felt it was a due change. https://github.com/ppy/osu/pull/21646 check this for more details on the matter.

Cant comment on the last two, but i assume theres likely something in the works, or will be in the future

**Comment by frenzibyte** [comment]
Quick retry not being instant is a known limitation and is tracked in https://github.com/ppy/osu/issues/9039.

**Comment by peppy** [comment]
Regarding this skin: It's hard to say why it's causing lag because there's so many inefficiencies in it. This will improve over time as we optimise systems, but in the mean time you're best off using a less bad skin.

If you really want to use the skin, try removing elements one by one. The health bar is probably a good start as it's 500+ frames of animation.
